src/Components/WebAssembly/WebAssembly.Authentication/src/Services/RemoteAuthenticationService.cs

```diff
@@ -67,11 +72,31 @@ public class RemoteAuthenticationService<
         IOptionsSnapshot<RemoteAuthenticationOptions<TProviderOptions>> options,
         NavigationManager navigation,
         AccountClaimsPrincipalFactory<TAccount> accountClaimsPrincipalFactory)
+        : this(jsRuntime, options, navigation, accountClaimsPrincipalFactory, null)
```

Comment:
My guess is that you're retaining this contructor overload (with logger being set to null) just to avoid the breaking change, or perhaps to keep people's unit tests working.

If so that's fine, and I can see that the logger isn't even used unless we get an incoming JS interop call, so people's unit tests won't be affected.

However, could we at least mark this constructor overload as `[Obsolete]` if there's no good reason for its continued existence in production? It would be nice to clear away this kind of thing eventually.

Reply:
There are a bunch of things like this that I have done to "avoid the breaking changes", so I will make sure I obsolete things too.

Reply:
I made the other constructor obsolete